sdk/identity/Azure.Identity/README.md

```diff
@@ -263,6 +263,12 @@ Not all credentials require this configuration. Credentials which authenticate t
 |`AZURE_USERNAME`|a username (usually an email address)
 |`AZURE_PASSWORD`|that user's password
 
+#### Managed identity
```

Comment:
Nit: Convert each subheading in this section to level 3.

Also, because of the way the sentence under the "Environment variables" heading reads, some customers may expect this table to apply to `EnvironmentCredential`. I assume this guidance is DAC-specific.

Reply:
How does this look?